lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -476,6 +476,20 @@ describe('AlfrescoViewerComponent', () => {
         //
     });
 
+    describe('originalMimeType', () => {
+
+        it('should set originalMimeType based on nodeData content', () => {
+            component.originalMimeType= 'application/pdf';
```

Comment:
This test seems still incorrect. You set 'application/pdf' to originalMimeType and then test in 'expect' if this is set based on nodeData content to 'application/pdf' but you already assigned 'application/pdf' to originalMimeType. So it does not prove that 'application/pdf' value comes from logic of ngOnInit and not because you set directly this value to originalMimeType. Please set different value to originalMimeType or keep originalMimeType as undefined and then expect will show if that field was set by ngOnInit.

Reply:
The test case has been rewritten